build/Dockerfile

```diff
@@ -58,6 +59,11 @@ ENV USER_NAME=psap-ci-runner \
 
 COPY . ${HOME}/
 
+RUN echo "psap-ci-runner:200000:1000" >> /etc/subuid \
+	&& echo "psap-ci-runner:200000:1000" >> /etc/subgid \
+	&& chmod 777 /usr/bin/newgidmap \
+	&& chmod 777 /usr/bin/newuidmap
```

Comment:
I don't follow what's happening here,

```
time="2022-01-02T01:29:52Z" level=warning msg="\"/\" is not a shared mount, this could cause issues or missing mounts with rootless containers"
Error: cannot setup namespace using newuidmap: fork/exec /usr/bin/newuidmap: operation not permitted
```

the files exist, otherwise `chmod` would crash, but the `chmod 777` is to get rid of the `operation not permitted` error?
I don't think it's a file permission error, otherwise it would fail with a message like this:
```
./mig-manager.log
zsh: permission denied: ./mig-manager.log
```
```
EPERM 1 Operation not permitted
# vs
EACCES 13 Permission denied
```

so must be some cgroup things being denied ...
Did you manage to reproduce the error in a local cluster?

Prow (OpenShift) sets some UID restrictions, see [this](https://stackoverflow.com/questions/68430367/openshift-is-running-all-pods-with-the-first-uid-from-the-range) and maybe [that](https://cloud.redhat.com/blog/a-guide-to-openshift-and-uids), I had issues around this topic during the early setup of the CI, but I'm not sure how to create Pods with such restrictions ... (let me know if you find out)